packages/@react-spectrum/dialog/chromatic/Dialog.chromatic.tsx

```diff
@@ -157,6 +158,105 @@ storiesOf('Dialog', module)
     }
   );
 
+storiesOf('Dialog/Popover', module)
+  .addParameters({chromaticProvider: {colorSchemes: ['light'], locales: ['en-US'], scales: ['medium'], disableAnimations: true}})
+  .add(
+    'position start default (12px) padding',
+    () => (
+      <View width="100%" overflow="auto">
+        <Grid
+          areas={[
+            'top    top',
+            'start  end',
+            'bottom bottom'
+          ]}
+          columns={['auto', 'auto']}
+          rows={['size-450', 'auto', 'size-450']}
+          height="100%"
+          marginTop="20px"
+          gap="size-100">
+          <View gridArea="start" justifySelf="start" alignSelf="center" paddingStart="20px">
+            <DialogTrigger type="popover" placement="bottom end" shouldFlip={false} isOpen>
+              <ActionButton>T</ActionButton>
+              <Dialog><Content>Placement Top</Content></Dialog>
+            </DialogTrigger>
+          </View>
+        </Grid>
+      </View>
+    )
+  )
+  .add(
+    'position start adjusted 6px padding',
+    () => (
+      <View width="calc(100% + 16px)" overflow="auto" margin="-8px">
+        <Grid
+          areas={[
+            'top    top',
+            'start  end',
+            'bottom bottom'
+          ]}
+          columns={['auto', 'auto']}
+          rows={['size-450', 'auto', 'size-450']}
+          height="100%"
+          marginTop="20px"
+          gap="size-100">
+          <View gridArea="start" justifySelf="start" alignSelf="center">
+            <DialogTrigger type="popover" placement="bottom end" shouldFlip={false} isOpen>
+              <ActionButton>T</ActionButton>
+              <Dialog><Content>Placement Top</Content></Dialog>
+            </DialogTrigger>
+          </View>
+        </Grid>
+      </View>
+    )
+  )
+  .add(
+    'position end adjusted 6px padding, no arrow',
+    () => (
+      <View width="calc(100% + 40px)" overflow="auto" margin="-20px">
+        <Grid
+          areas={[
+            'top    top',
+            'start  end',
+            'bottom bottom'
+          ]}
+          columns={['auto', 'auto']}
+          rows={['size-450', 'auto', 'size-450']}
+          height="100%"
+          marginTop="20px"
+          gap="size-100">
+          <View gridArea="end" justifySelf="end" alignSelf="center">
+            <DialogTrigger type="popover" shouldFlip={false} isOpen>
+              <ActionButton>T</ActionButton>
+              <Dialog><Content>Placement Top</Content></Dialog>
+            </DialogTrigger>
+          </View>
+        </Grid>
+      </View>
+    )
+  )
+  .add(
+    'position bottom adjusted 6px padding',
```

Comment:
This story is a little wonky. The bottom case was the hardest to get right so I wanted this example. Ant it uses `window.innerHeight` instead of `document.body.getBoundingClientRect()`. Due to the story parent min-height of 1000px the button and popover are forced out of the easily viewable area and don't really line up in the snapshot. Running Chromatic Storybook locally the popover shows the adjusted 6px padding, but in their snapshot you can't see that. The arrow is gone, maybe that is enough?

I tried a height of 100% instead of the minHeight=1000px and it still doesn't work how I'd like in the snapshot. Might just do a top story instead of this bottom one.